README.md

```diff
@@ -232,6 +232,44 @@ Display the streamed video:
 gst-launch-1.0 -v udpsrc port=5600 caps='application/x-rtp, media=(string)video, clock-rate=(int)90000, encoding-name=(string)H264' ! rtph264depay ! avdec_h264 ! videoconvert ! autovideosink sync=false
 ```
 
+View the streamed camera frames in OGC:
+
+`Open QGC > Application Settings > Video Settings > Select UDP h.264 Video Stream & use port 5600`
+
+### 4. Using Gimbal
+
+To control gimbal using RC channels,
```

Comment:
Suggestion: The Iris model is equipped with a 3d gimbal and camera that can be controlled directly in MAVProxy using the RC overrides.